src/components/Icons/Icon/Icon.test.tsx

```diff
@@ -0,0 +1,23 @@
+import { render, fireEvent } from "@testing-library/react";
+import "@testing-library/jest-dom";
+import { FiSun } from "react-icons/fi";
+import { Icon } from "./Icon";
+
+describe("Icon", () => {
+  it("renders the correct icon and calls onClick function when clicked", () => {
+    const onClickMock = jest.fn();
+    const { getByTestId } = render(
+      <Icon
+        src={<FiSun />}
+        color="#1AE19D"
+        size="1em"
+        onClick={onClickMock}
+        className=""
```

Comment:
here and also in the story you pass an empty string, is there a way to omit this prop by change to `?string` in `IconProps`

Reply:
noted